### PR TITLE
Feature/rdsssam 132 create new basic samvera rdsscdm work type

### PR DESCRIPTION
**Pull request by fmcc**

### CODE REVIEW

File: willow/app/models/concerns/hyrax/serializers.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module Hyrax
+  module Serializers
+    def to_s
+      if title.present?
+        if title.is_a? String
```
Comment (blackrat) [inline willow/app/models/concerns/hyrax/serializers.rb]: Ugh. Do we have a contract that says that title can be empty, a string or an array. I've traced through several layers of code and can see either of those are possible, but they seem to be always folded to an array. I'd probably have a delegated title and label delegation and use I18n for the text value in preference. Also, a mix in this generic seems to be a little too powerful.

File: willow/app/models/solr_document.rb [hunk 2]
```diff
@@ -26,6 +25,26 @@ class SolrDocument
 
   use_extension( Hydra::ContentNegotiation )
 
+  # RDSS CDM additions:
+  def title
+    self[Solrizer.solr_name('title', :stored_searchable)]
```
Comment (blackrat) [inline willow/app/models/solr_document.rb]: Ok. So here we are using single quotes, so there isn't a consistency. I'ld prefer to use these always when we don't have any interpolation.

File: willow/config/locales/en.yml [hunk 3]
```diff
@@ -1,6 +1,13 @@
 en:
   willow:
     fields:
+      # RDSS CDM additions:
+      title: 'Title'
```
Comment (blackrat) [inline willow/config/locales/en.yml]: You don't need to quote strings in an internationalisation yaml file.

File: willow/config/locales/hyrax.en.yml [hunk 4]
```diff
@@ -27,6 +27,10 @@ en:
           rights_tesim: Rights
           subject_tesim: Subject
         show:
+          title: Title
```
Comment (blackrat) [inline willow/config/locales/hyrax.en.yml]: Why are these duplicated? You should be able to refer to them in a single place if desired.

File: willow/config/locales/rdss_cdm.en.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    icons:
+      rdss_cdm:     'fa fa-file-text-o'
+    select_type:
+      rdss_cdm:
+        name:               "Rdss Cdm"
+        description:        "Rdss cdm works"
+  simple_form:
+    labels:
+      rdss_cdm:
+        object_description: 'Description'
```
Comment (blackrat) [inline willow/config/locales/rdss_cdm.en.yml]: Eek. And again. Is this what Danny was talking about when he said it wasn't DRY? I did wonder at that comment.

File: willow/app/models/concerns/hyrax/serializers.rb [hunk 6]
```diff
@@ -0,0 +1,17 @@
+module Hyrax
+  module Serializers
+    def to_s
+      if title.present?
```
Comment (blackrat) [inline willow/app/models/concerns/hyrax/serializers.rb]: Do we really want to use the rails '.present?' method or stick to pure Ruby approaches?

File: willow/app/controllers/concerns/hyrax/notifications/notifiers.rb [hunk 7]
```diff
@@ -11,15 +11,23 @@ module Notifiers
...
 
         def after_update_response
-          # only send METADATA_UPDATE message if the object is active
-          if curation_concern.state.id == 'http://fedora.info/definitions/1/0/access/ObjState#active'
-            ActiveSupport::Notifications.instrument(Events::METADATA_UPDATE, {curation_concern_type: self.class.curation_concern_type, object: curation_concern})
+          # We'll maintain the later behaviour for other Work types, but skip it for the new type until 
+          # the correct CRUD behaviour is implemented. 
+          unless curation_concern.instance_of? RdssCdm
+            # only send METADATA_UPDATE message if the object is active
+            if curation_concern.state.id == 'http://fedora.info/definitions/1/0/access/ObjState#active'
```
[the remark this answers is not in the document]
Comment (fmcc) [reply willow/app/controllers/concerns/hyrax/notifications/notifiers.rb]: Agreed - this all needs to be overhauled as part of RDSSSAM-68, but it was out of scope for this ticket (and it didn't seem sensible to head down that rabbit hole just now).

File: willow/app/controllers/catalog_controller.rb [hunk 8]
```diff
@@ -31,7 +31,16 @@ def self.modified_field
     config.default_solr_params = {
       qt: "search",
       rows: 10,
-      qf: "title_tesim description_tesim creator_tesim keyword_tesim"
+      qf: [
+        solr_name("title", :stored_searchable),
```
[the remark this answers is not in the document]
Comment (fmcc) [reply willow/app/controllers/catalog_controller.rb]: If you could run a job to fix this globally that'd be good. I'd rather adopt sensible conventions.

File: willow/config/locales/rdss_cdm.es.yml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+es:
+  hyrax:
+    icons:
+      rdss_cdm:     'fa fa-file-text-o'
+    select_type:
+      rdss_cdm:
+        # TODO: translate `human_name` into Spanish
```
Comment (tripledub) [inline willow/config/locales/rdss_cdm.es.yml]: `Nombre Humano`

**Pull request merged**